commands/create/cluster/v5.go

```diff
@@ -88,8 +123,16 @@ func addClusterV5(def *types.ClusterDefinitionV5, args Arguments, clientWrapper
 		return "", true, microerror.Mask(errors.ClusterOwnerMissingError)
 	}
 
+	if def.MasterNodes != nil && def.ReleaseVersion != "" && !haMastersFeature.HAMasters.IsSupported(config.Config.Provider, def.ReleaseVersion) {
```

Comment:
If the user set a release version that is not supported, and also set HA Master, throw an error